src/hadoop-jobhistory/deploy/start.sh

```diff
@@ -28,14 +28,14 @@ pushd $(dirname "$0") > /dev/null
 # Hadoop jobhistory
 kubectl apply --overwrite=true -f hadoop-jobhistory.yaml || exit $?
 
-PYTHONPATH="../.." python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k jobhistory -v "true"
+PYTHONPATH="../../../deployment" python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k jobhistory -v "true"
```

Comment:
@YitongFeng  To ensure, whether this line could be removed or not.

Reply:
could removed